yogstation/code/game/gamemodes/gangs/gang_pen.dm

```diff
@@ -0,0 +1,60 @@
+/*
+ * Gang Boss Pens
+ */
+/obj/item/pen/gang
+	var/cooldown
+	var/last_used
+
+/obj/item/pen/gang/Initialize()
+	last_used = world.time
```

Comment:
last_used isn't used anywhere, please remove it.

Reply:
mk